clients/privacy-center/cypress/fixtures/consent/test_banner_options.json

```diff
@@ -102,6 +102,7 @@
     "isGeolocationEnabled": false,
     "geolocationApiUrl": "",
     "privacyCenterUrl": "http://localhost:3000",
-    "fidesApiUrl": "http://localhost:8080/api/v1"
+    "fidesApiUrl": "http://localhost:8080/api/v1",
+    "fidesTcfGdprApplies": true
```

Comment:
since this is an option that defaults to true, and because our test harness doesn't go through `fides-js` the API (which defaults this value to true), we set the default to true here.